lib/implore.es6.js

```diff
@@ -122,6 +122,17 @@ export default function implore(request, callback) {
 			response.status = xhr.status;
 			response.type = xhr.getResponseHeader('Content-Type');
 
+			// Expose headers in raw form (string) and in a more convenient form (hash)
+			response.rawHeaders = xhr.getAllResponseHeaders();
+			const responseHeaders = response.rawHeaders.split('\n');
+			response.headers = {};
+			for (let header of responseHeaders) {
+				const parts = header.split(':');
```

Comment:
Do you mean L131 with the if statement?

I added that in response to an error with one of the requests in the app that uses fluxthis. I think it's when headers ends with a blank line.